### DISCUSSION TITLE
external hdd

### DISCUSSION THREAD

**Discussion by bsoener** [jlesage/docker-jdownloader-2#195] [Q&A] [answered]
Hi folks. I've been messing around with my portainer for days and can't get it to work. I have a Raspi 5 and use the system dietpi with a sd card. I have an Argon one for my case. In this case I have installed a Nvme and is already mounted. Unfortunately I can't manage to integrate the volume via portainer the jdownloader. How do I manage to integrate my nvme. I don't want to delete the nvme because my movie collection is on it and I also have access via samba. I would be really happy if someone could help me.

**Comment by bsoener** [comment]
Can't anyone help me with this? I have been trying for days and unfortunately have not been successful.

**Comment by jlesage** [comment]
You need to map a directory from the host to a directory inside the container.  This is done by adding a "volume mapping".   This might help: https://docs.portainer.io/user/docker/containers/advanced#volumes

**Comment by bsoener** [comment]
Thanks for the information. I already know that I can select a volume, but don't I have to create a volume beforehand? I have problems with that. I can't find my external hard disk and I don't know how to insert it. It is mounted and contains data. I would appreciate a little tip

**Reply by mr-manuel** [reply]
You can simply select a directory and mount that directory in the container.

1. You write that you have already mounted the volume and you are able to access the data. I which directory is the data located?
2. Do you use Docker Compose or did you create the container directly in Portainer?

**Comment by bsoener** [comment]
I have mounted the hard disk directly on the dietpi. Jdownloader runs via Docker. However, I do not see my hard disk when creating a volume. As soon as I want to create a volume, it is created on my sd card. I am currently at work. As soon as I get home I can take pictures.

**Comment by bsoener** [comment]
![IMG_9413](https://github.com/jlesage/docker-jdownloader-2/assets/77502899/67b169e3-167d-4982-8783-882c679721f7)
![IMG_9412](https://github.com/jlesage/docker-jdownloader-2/assets/77502899/c0e2bd43-c9c0-4d38-9806-70889f8345f1)
![IMG_9411](https://github.com/jlesage/docker-jdownloader-2/assets/77502899/e52cee13-3e8e-46fb-8fd1-651077b41087)

**Reply by bsoener** [reply]
Unfortunately I can't find the path for my external one here. 
My external is listed in: nvme0n1 and path is mnt/filme. I'm sorry but I'm really struggling with this right now

**Reply by mr-manuel** [reply, marked as answer]
This is completely wrong configured.

container: is the path in the container

volume: is a docker volume on the host. `portainer_data` is the volume where portainer saves the data

Remove the `/mnt/filme` and change the `/output` from volume to binding and insert the binding path `/mnt/file`, if this is the directory of the mounted disk on the host.

**Comment by bsoener** [comment]
You are great! It's very simple, but I really had a mistake in my thinking. Thank you very much for your support. Now it works and I've done a test. I'm so happy.
